cmd/bridge/main.go

```diff
@@ -499,12 +502,36 @@ func main() {
 		Handler: srv.HTTPHandler(),
 	}
 
-	log.Infof("Binding to %s...", httpsrv.Addr)
+	klog.V(2).Infof("Binding to %s...", httpsrv.Addr)
 	if listenURL.Scheme == "https" {
-		log.Info("using TLS")
-		log.Fatal(httpsrv.ListenAndServeTLS(*fTlSCertFile, *fTlSKeyFile))
+		klog.V(2).Info("using TLS")
+		klog.Fatal(httpsrv.ListenAndServeTLS(*fTlSCertFile, *fTlSKeyFile))
 	} else {
-		log.Info("not using TLS")
-		log.Fatal(httpsrv.ListenAndServe())
+		klog.V(2).Info("not using TLS")
+		klog.Fatal(httpsrv.ListenAndServe())
 	}
 }
+
+// initLogs initializes logs the way we want for kubernetes.
+func initLogs() {
+	klog.InitFlags(nil)
+	log.SetOutput(KlogWriter{})
```

Comment:
I'm not sure why you want to import both `klog` & `log`.  I think we only need `klog`.